apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -60,12 +60,9 @@ function get_wpcom_redirect_to() {
  * @return string The login URL
  */
 function get_login_url() {
-	// If we're on WPCOM use a WordPress.com login URL.
-	if ( function_exists( 'localized_wpcom_url' ) ) {
-		return localized_wpcom_url( '//wordpress.com/log-in?redirect_to=' . get_wpcom_redirect_to() );
-	}
+	$default_url = site_url() . '/wp-login.php?redirect_to=' . set_url_scheme( original_request_url() );
 
-	return site_url() . '/wp-login.php?redirect_to=' . set_url_scheme( original_request_url() );
+	return apply_filters( 'wpcom_public_coming_soon_login_url', $default_url, get_wpcom_redirect_to() );
```

Comment:
The WPCOM filter ignores `$default_url` and returns the WPCOM login url + redirect